### PR TITLE
Some more language fixes I run into when going through the exercises

### PR DESCRIPTION
**Pull request by AnnikaLau**

### CODE REVIEW

**Comment by mathause** [comment]
Thanks for all the language fixes!

File: Part0_Introduction/ex0_1_jupyter_notebook.ipynb [hunk 1]
```diff
@@ -342,7 +342,7 @@
     " * Press <kbd>Tab</kbd> to expand all available functions: `np.<tab>`\n",
     " * Add a `?` to any function to get the help: `np.mean?`\n",
     " * Add `??` to any function to get the source code of the function (includes the help): `np.mean??`\n",
-    " * Use <kbd>Shift</kbd><kbd>Tab</kbd>\n",
+    " * Use <kbd>Shift</kbd><kbd>Tab</kbd>\n" to get information about a function,
```
Comment (mathause) [inline Part0_Introduction/ex0_1_jupyter_notebook.ipynb:345]:
```suggestion
    " * Use <kbd>Shift</kbd><kbd>Tab</kbd>\n" to get information about a function",
```
?

**Comment by AnnikaLau** [comment]
@mathause : feel free to add any other language fixes to this pull request

**Comment by chadhat** [comment]
@AnnikaLau Can we merge this branch by tomorrow morning? So that we can update the jupyter hub.

**Comment by AnnikaLau** [comment]
@chadhat: I was planning to go through all exercises again today and would add anything to this pull request, which I stumble over. Could you update to the current version on jupyter hub and update again later in the evening today? There shouldn't be any essential changes in this pull request.

**Comment by chadhat** [comment]
@AnnikaLau we will update the jupyter hub setup. I can not promise that we can update it again this evening but as you said, these changes are not really critical.

**Comment by chadhat** [comment]
@AnnikaLau Thanks for the improvements. Since these are language fixes, I just glanced at a few of the changes. I assume that everything is fine and merge the branch

**Comment by AnnikaLau** [comment]
Thank you!

**Pull request merged**